ci/test.yml

```diff
@@ -150,7 +150,12 @@ rollout manual:
     - yarn install --pure-lockfile --cache-folder .yarn --prefer-offline
     - yarn workspace @trezor/utils build:lib
     - yarn workspace @trezor/transport build:lib
+    # run e2e tests
     - ./docker/docker-transport-test.sh
+    # test protobuf update script
+    - yarn workspace @trezor/transport update:protobuf
+    # todo: not sure if it should go here.
+    - yarn workspace @trezor/transport type-check
```

Comment:
todo: is there a better place in CI to call this?

Reply:
type-check is called in `validation` stage, why call it again here?

Reply:
actually a good question. maybe these lines won't be needed. my intention was to test the output of `yarn workspace @trezor/transport update:protobuf` script, which is some d.ts file. This d.ts file should be commited into git anyway, so yes, it should be checked by `validation` stage.
It would make more sense here to checke whther `yarn workspace @trezor/transport update:protobuf` did not produce any unstaged changes and exit with error if yes.